test/common/io/io_uring_worker_impl_integration_test.cc

```diff
@@ -463,21 +461,24 @@ TEST_F(IoUringWorkerIntegraionTest, MergeInjection) {
   EXPECT_EQ(socket.nr_completion_, 1);
 }
 
-TEST_F(IoUringWorkerIntegraionTest, AcceptSocketBasic) {
+TEST_F(IoUringWorkerIntegraionTest, AcceptSocketAccept) {
   initialize();
   socket(false, true);
   listen();
 
   auto& socket = io_uring_worker_->addAcceptSocket(listen_socket_, io_uring_handler_);
   EXPECT_EQ(io_uring_worker_->getSockets().size(), 1);
+
   connect();
 
-  dispatcher_->run(Event::Dispatcher::RunType::NonBlock);
-  EXPECT_TRUE(SOCKET_VALID(io_uring_handler_.server_socket_fd_));
+  while (io_uring_handler_.server_socket_fd_ == -1) {
```

Comment:
I removed the loop previously, since I think it must be triggered in the next event loop.